vault/core.go

```diff
@@ -2094,6 +2095,16 @@ func (c *Core) emitMetrics(stopCh chan struct{}) {
 				}
 			}
 			c.stateLock.RUnlock()
+		case <-identityCountTimer:
+			go func() {
+				ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+				defer cancel()
+				entities, err := c.countActiveEntities(ctx)
+				if err != nil {
+					c.logger.Error("error counting identity entities", "err", err)
+				}
+				metrics.SetGauge([]string{"identity", "num_entities"}, float32(entities.Entities.Total))
```

Comment:
Please move the metrics.SetGauge into an else clause so it's not sent when there's an error.

Other than that, looks good!

Reply:
Oops, of course. Updated.